src/workloads/query/TenMDocCollection_IntId.yml

```diff
@@ -111,9 +111,9 @@ Actors:
   - ActorFromTemplate:
       TemplateName: CrudTemplate
       TemplateParameters:
-        Name: IDFieldPointProject128Threads
+        Name: IDHackEqPointProject128Threads
         Find:
-          # Using $eq turns this from an IDHACK find into a scan find.
+          # Queries of the shape {_id: {$eq: <value>}} are eligible for IDHACK.
           Filter: {_id: {$eq: {^RandomInt: {min: 0, max: 9999999}}}}
```

Comment:
Since this query now uses IDHACK, we no longer have a query on _id that doesn't use IDHACK in this workload. Should I add one? One idea would just be to have a query with an additional predicate, something like
`{_id: {$eq: <random int>, $lt: <random int>}}`

Reply:
I think this is something the owner of the workload should answer (@mongodb/query).

Reply:
As discussed offline with @cswanson310 I won't be adding another query here. We will be benchmarking the optimizer via https://jira.mongodb.org/browse/SPM-2658, which should include range predicate index scans. @miguelangelnieto do you have any other comments about this PR?